classes/robot/crawler.php

```diff
@@ -258,23 +258,25 @@ public function reset_for_recrawl($nodeid) {
             // Mark all nodes that link to this as needing a recrawl.
             if ($DB->get_dbfamily() == 'mysql') {
                 $DB->execute("UPDATE {tool_crawler_url} u
-                         INNER JOIN {tool_crawler_edge} e ON e.a = u.id
-                         SET needscrawl = ?,
-                                 lastcrawled = null
-                         WHERE e.b = ?", [$time, $nodeid]);
+                              INNER JOIN {tool_crawler_edge} e ON e.a = u.id
```

Comment:
indenting is better, but still weird, looks 4 spaces deeper than it should be?

Reply:
Should be fixed now